protocols/smtp/src/test/java/org/apache/james/protocols/smtp/AbstractSMTPServerTest.java

```diff
@@ -493,16 +494,16 @@ public void testHeloEnforcementDisabled() throws Exception {
     public void testHeloHookPermanentError() throws Exception {
         HeloHook hook = new HeloHook() {
 
-            @Override
-            public void init(Configuration config) throws ConfigurationException {
-
-            }
-
-            @Override
-            public void destroy() {
-
-            }
-
+            @Override

```

Comment:
Line breaks changes (to default) while saving.
This is not shown by GitHub, and should be ignore by GitHub diff, IMHO.